service/internal/integrationtest/s3/checksum_test.go

```diff
@@ -541,6 +541,38 @@ func TestInteg_RequireChecksum(t *testing.T) {
 	}
 }
 
+func TestInteg_RequireChecksumWithoutRequestAlgorithmMember(t *testing.T) {
+	params := &s3.PutBucketOwnershipControlsInput{
+		Bucket: &setupMetadata.Buckets.Source.Name,
+		OwnershipControls: &types.OwnershipControls{
+			Rules: []types.OwnershipControlsRule{
+				{
+					ObjectOwnership: types.ObjectOwnershipBucketOwnerPreferred,
+				},
+			},
+		},
+	}
+
+	t.Logf("putting bucket ownership control: %q", *params.Bucket)
+	result, err := s3client.PutBucketOwnershipControls(context.Background(), params)
+	if err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+	computedChecksums, ok := s3.GetComputedInputChecksumsMetadata(result.ResultMetadata)
+	if !ok {
+		t.Fatal("expect computed checksums metadata present, got nil")
+	}
+
+	expectComputedChecksums := s3.ComputedInputChecksumsMetadata{
+		ComputedChecksums: map[string]string{
+			"CRC32": "cK9COg==",
+		},
+	}
+	if diff := cmpDiff(expectComputedChecksums, computedChecksums); diff != "" {
+		t.Errorf("expect computed checksum metadata match\n%s", diff)
```

Comment:
`\n%s`  will print a new line and then the result. It should be `%s\n`

Reply:
I guess the old test case did that to make diff clear? But will change that